pom.xml

```
@@ -43,6 +43,7 @@
     <gt-version>30.0</gt-version>
     <pdfbox-version>2.0.26</pdfbox-version>
     <metrics-version>4.2.12</metrics-version>
+    <jackson-dataformat-yaml.version>2.16.1</jackson-dataformat-yaml.version>
```

this should preferably match the Jackson 2 version in use by GeoServer, currently 2.17.1, see: 
https://github.com/geoserver/geoserver/blob/3c5fa6df3b5a57d459dd902a3f67dee790ab0918/src/pom.xml#L139-L141

```suggestion
    <jackson-dataformat-yaml.version>2.17.1</jackson-dataformat-yaml.version>
```

to prevent the possibility of ending up in transitive dependency hell

---

Also note https://github.com/geoserver/geoserver/pull/7767 that upgrades GeoServer to Jackson 2.17.2

---

Done thanks.